ccdproc/core.py

```diff
@@ -1558,6 +1576,151 @@ def cosmicray_median(ccd, error_image=None, thresh=5, mbox=11, gbox=0,
         raise TypeError('ccd is not an numpy.ndarray or a CCDData object.')
 
 
+def ccdmask(ratio, findbadcolumns=False, byblocks=False, ncmed=7, nlmed=7,
+            ncsig=15, nlsig=15, lsigma=9, hsigma=9, ngood=5):
+    '''
+    Uses method based on the IRAF ccdmask task to generate a mask based on the
+    given input.
+
+    Parameters
+    ----------
+    ratio : `~ccdproc.CCDData`
+        Data to used to form mask.  Typically this is the ratio of two flat
+        field images.
+
+    findbadcolumns: `bool`, optional
+        If set to True, the code will search for bad column sections.  Note that
+        this treats columns as special and breaks symmetry between lines and
+        columns and so is likely only appropriate for detectors which have
+        readout directions.
+
+    byblocks : `bool`, optional
+        If set to true, the code will divide the image up in to blocks of size
+        nlsig by ncsig and determine the standard deviation estimate in each
+        block (as described in the original IRAF task, see Notes below).  If
+        set to False, then the code will use `percentile_filter` in 
```

Comment:
Instead of saying "... in scipy.ndimage" it should link directly to `scipy`:

```
`scipy.ndimage.percentile_filter`
```

Otherwise the sphinx build fails.